service/provider/src/main/java/gov/va/vro/service/provider/bip/service/BipApiService.java

```diff
@@ -2,17 +2,7 @@
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import gov.va.vro.model.bip.BipClaim;
-import gov.va.vro.model.bip.BipClaimResp;
-import gov.va.vro.model.bip.BipContentionResp;
-import gov.va.vro.model.bip.BipFileUploadPayload;
-import gov.va.vro.model.bip.BipFileUploadResp;
-import gov.va.vro.model.bip.BipUpdateClaimResp;
-import gov.va.vro.model.bip.ClaimContention;
-import gov.va.vro.model.bip.ClaimStatus;
-import gov.va.vro.model.bip.CreateContentionReq;
-import gov.va.vro.model.bip.FileIdType;
-import gov.va.vro.model.bip.UpdateContentionReq;
+import gov.va.vro.model.bip.*;
```

Comment:
no "*". That is a style check and also clarified what is added with this log.

Reply:
done